plugins/gen-design-doc/aac_gen_design_doc/tests/test_aac_gen_design_doc_impl.py

```diff
@@ -0,0 +1,9 @@
+from unittest import TestCase
```

Comment:
I added this file as a placeholder. If there aren't any tests, we should remove this file. That being said, a few tests might not hurt.

Reply:
I'm adding tests. 👍